lighter/system.py

```diff
@@ -361,6 +361,36 @@ def setup(self, stage: str) -> None:
             self.predict_dataloader = partial(self._base_dataloader, mode="predict")
             self.predict_step = partial(self._base_step, mode="predict")
 
+    @property
```

Comment:
Can we reduce the docs for this? Especially the `Raises` 

This feels like an "extra" to system so don't want it taking up too much space.

Reply:
better?